_locales/nl_nl.json

```diff
@@ -283,11 +283,11 @@
     "SETTINGS/AUTODOWNLOAD/delay/form": "Vertraging:",
     "SETTINGS/AUTODOWNLOAD/delay/hdr": "Auto-Download vertraging",
     "SETTINGS/AUTODOWNLOAD/disable/btn": "Niet automatisch naar torrents zoeken",
-    "SETTINGS/AUTODOWNLOAD/disable/desc": "Als je op de knop hieronder klikt wordt het automatische download proces voor afleveringen gestopt.",
+    "SETTINGS/AUTODOWNLOAD/disable/desc": "Als je op de knop hieronder drukt, wordt het automatische downloadproces voor afleveringen gestopt.",
     "SETTINGS/AUTODOWNLOAD/enable/btn": "Automatisch naar torrents zoeken",
-    "SETTINGS/AUTODOWNLOAD/enable/desc": "Door te klikken op de onderstaande knop start een automatische controle op torrents voor afleveringen uitgezonden zijn",
-    "SETTINGS/AUTODOWNLOAD/enable/desc2": "(voor series die niet al gedownload of gezien zijn).\nHet zorgt ervoor dat er minstens ",
-    "SETTINGS/AUTODOWNLOAD/enable/desc3": " seeders voor de torrent en automatisch de torrent kwaliteit die je ingesteld hebt.",
+    "SETTINGS/AUTODOWNLOAD/enable/desc": "Als je op onderstaande knop drukt, start er elke 15 minuten een automatische check, die torrents download voor uitgezonden afleveringen binnen ",
+    "SETTINGS/AUTODOWNLOAD/enable/desc2": " dagen (welke nog niet gedownload of gemarkeerd als bekeken zijn).\n Het zorgt ervoor dat er minstens 
+    "SETTINGS/AUTODOWNLOAD/enable/desc3": " seeders voor de torrent zijn en gebruikt automatisch jouw voorkeuren voor de resolutie van de torrent.",
```

Comment:
oops, looks like you dropped a quote and comma at the end of this line ;-)

Reply:
Fixed in newest version. Thanks for noticing!